.rubocop.yml

```diff
@@ -205,6 +205,11 @@ Style/TrivialAccessors:
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#attr_family'
   Enabled: false
 
+Style/Lambda:
+  Description: 'Use the `lambda` method for multiline lambdas'
+  StyleGuide: 'https://github.com/rubocop-hq/ruby-style-guide#lambda-multi-line'
+  Enabled: false
```

Comment:
For whatever reason
```ruby
  scope :two_months_data, do |hash|
    where(followable_id: hash[:followable_id], followable_type: hash[:followable_type]).
      where("created_at > ?", 2.months.ago)
   end
```

was not working with Rails. I couldn't figure out the syntax, and any other way seem to look too messy so I just turned this cop off.